Makefile

```diff
@@ -21,9 +21,31 @@ clean:
 	rm -rf */*/__pycache__
 	rm -rf */*/*/__pycache__
 
-install:
+.PHONY: build install
+
+build:
+	@echo "BUILDING PYTHON PACKAGE..."
+	python3 setup-partial.py build
+	@echo "BUILDING TRANSLATIONS..."
+	for LANG in `cat po/LINGUAS`; do \
+	echo $$LANG; \
+	mkdir -p build/$$LANG; \
+	msgfmt po/$$LANG.po -o build/$$LANG/catapult.mo; \
+	done
+	@echo "BUILDING DESKTOP FILE..."
+	mkdir -p build
+	msgfmt --desktop -d po \
+	--template data/io.otsaloma.catapult.desktop.in \
+	-o build/io.otsaloma.catapult.desktop
+	@echo "BUILDING APPDATA FILE..."
+	mkdir -p build
+	msgfmt --xml -d po \
+	--template data/io.otsaloma.catapult.appdata.xml.in \
+	-o build/io.otsaloma.catapult.appdata.xml
+
+install: build
 	@echo "INSTALLING PYTHON PACKAGE..."
-	./setup-partial.py install $(if $(DESTDIR),--root=$(DESTDIR),) --prefix=$(PREFIX)
+	python3 setup-partial.py install $(if $(DESTDIR),--root=$(DESTDIR),) --prefix=$(PREFIX)
```

Comment:
Why this change?

Reply:
Must have been some left-over from previous version.